apps/api/src/app/widgets/usecases/get-notifications-feed/get-notifications-feed.usecase.ts

```diff
@@ -14,7 +15,9 @@ export class GetNotificationsFeed {
     private subscriberRepository: SubscriberRepository
   ) {}
 
-  async execute(command: GetNotificationsFeedCommand): Promise<MessageEntity[]> {
+  async execute(command: GetNotificationsFeedCommand): Promise<MessagesResponseDto> {
+    const limit = 10;
```

Comment:
```suggestion
    const LIMIT = 10;
```
To remark that is a constant. Usually per convention.

Reply:
done